### PR TITLE
add docker support starter

### PR DESCRIPTION
**Pull request by abdoutelb**
Included a `docker-compose` file for dev & production

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+COPY package.json ./
+
+RUN yarn install
+
+COPY . .
+
+EXPOSE 3000
+
+CMD ["npm","start"]
```
Comment (netcode) [inline Dockerfile:16]: Can you please use yarn instead of npm ? Just to unify the tools used.

**Review by netcode — CHANGES_REQUESTED** [review]

**Comment by netcode** [comment]
@abdoutelb Thank you for your contribution, Can you make the change to use yarn in the docker file ?

**Comment by abdoutelb** [comment]
> @abdoutelb Thank you for your contribution, Can you make the change to use yarn in the docker file ?

Sure Updated with readme instructions

**Review by netcode — APPROVED** [review]

**Comment by netcode** [comment]
Merged. Thanks.

**Pull request merged**